bindings/go/configuration/extract/v1alpha1/config.go

```diff
@@ -0,0 +1,91 @@
+package v1alpha1
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	v1 "ocm.software/open-component-model/bindings/go/configuration/v1"
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+const (
+	// ConfigType defines the type identifier for transformation configurations
+	ConfigType = "extract.oci.artifact.ocm.software"
+)
+
+var scheme = runtime.NewScheme()
+
+func init() {
+	scheme.MustRegisterWithAlias(&Config{}, runtime.NewVersionedType(ConfigType, Version))
+}
+
+// Config represents the top-level configuration for the transformation.
```

Comment:
I dont understand this pr, now this is for transformation, the config type doesnt fit and we have the Temp Folder in here which is already in another config